AUTHORS.txt

```diff
@@ -41,3 +41,4 @@ Nicola Corna
 Robert Eckelmann
 Sam Condon
 Andreas Maeder
+Bastian Leykauf
```

Comment:
Yes, I have forked and branched before I was added to the author file. I still don't really get why there is a conflict, as the files look the same. Anyway, this can be resolved upon merge.

Reply:
I could suggest to visit you branch dev/pendulum-cnt91 with web browser and click on fetch upstream to realign to master changes.
![Screenshot_20211104-200306_Chrome](https://user-images.githubusercontent.com/37845722/140404054-b53ce9aa-db71-4cd2-9367-0b6297e5fe15.jpg)

Reply:
Thanks. I actually did that to my forked master branch. I thought I then integrated these changes into `bleykauf:dev/pendulum-cnt91` (where I also squashed some changes from an intermediate branch to not have too many commits with unverified code (did a lot from home without the actual device present).

I already have a new device in the pipeline, so I will try to get it right this time.

Thank you for all the feedback, it's really great contributing to this project :)